webapp/pages/terms-and-conditions.vue

```diff
@@ -1,36 +1,85 @@
 <template>
   <div>
     <ds-space>
-      <ds-heading tag="h2">{{ $t('site.termsAndConditions') }}</ds-heading>
+      <ds-heading tag="h2">{{ $t(`site.termsAndConditions`) }}</ds-heading>
     </ds-space>
     <ds-container>
+      <no-ssr>
+        <div>
+          <ol>
+            <li v-for="section in sections" :key="section">
+              <strong>{{ $t(`termsAndConditions.${section}.title`) }}:</strong>
+              <p v-html="$t(`termsAndConditions.${section}.description`)" />
+            </li>
+          </ol>
+          <p>{{ $t(`termsAndConditions.have-fun`) }}</p>
+          <br />
+          <p>
+            <strong v-html="$t(`termsAndConditions.closing`)" />
+          </p>
+        </div>
+      </no-ssr>
       <div>
-        <ol>
-          <li v-for="section in sections" :key="section">
-            <strong>{{ $t(`termsAndConditions.${section}.title`) }}:</strong>
-            <p v-html="$t(`termsAndConditions.${section}.description`)" />
-          </li>
-        </ol>
-        <p>{{ $t('termsAndConditions.have-fun') }}</p>
-        <br />
-        <p>
-          <strong v-html="$t('termsAndConditions.closing')" />
-        </p>
+        <ds-modal
+          v-if="isOpen"
+          v-model="isOpen"
+          v-title="$t(`site.termsAndConditionsNewTitle`)"
+          force
+          extended
+          confirm-label="Ich habe es gelesen und verstanden"
+          cancel-label="abbrechen"
+          v-on:confirm="setNewConfirmeVersion"
+        >
+          <div>
+            <ol>
+              <li v-for="section in sections" :key="section">
+                <strong>{{ $t(`termsAndConditions.${section}.title`) }}:</strong>
+                <p v-html="$t(`termsAndConditions.${section}.description`)" />
+              </li>
+            </ol>
+            <p>{{ $t(`termsAndConditions.have-fun`) }}</p>
+            <br />
+            <p>
+              <strong v-html="$t(`termsAndConditions.closing`)" />
+            </p>
+          </div>
+        </ds-modal>
+        <div v-if="isLoggedIn">
+          <ds-button primary icon="rocket" class="display:none" @click="isOpen = true">
+            {{ $t(`site.termsAndConditionsNewConfirm`) }}
+          </ds-button>
+        </div>
       </div>
     </ds-container>
   </div>
 </template>
 
 <script>
+import { mapGetters, mapActions } from 'vuex'
+export const VERSION = '0.0.3'
+
 export default {
   layout: 'default',
   head() {
     return {
       title: this.$t('site.termsAndConditions'),
     }
   },
+  computed: {
+    ...mapGetters({
+      isLoggedIn: 'auth/isLoggedIn',
+    }),
+  },
+  mounted() {
+    if (this.isLoggedIn) {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-16T08:55:37Z" title="Friday, August 16th 2019, 10:55:37 am +02:00">Aug 16, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,36 +1,85 @@
 <template>
   <div>
     <ds-space>
-      <ds-heading tag="h2">{{ $t('site.termsAndConditions') }}</ds-heading>
+      <ds-heading tag="h2">{{ $t(`site.termsAndConditions`) }}</ds-heading>
     </ds-space>
     <ds-container>
+      <no-ssr>
+        <div>
+          <ol>
+            <li v-for="section in sections" :key="section">
+              <strong>{{ $t(`termsAndConditions.${section}.title`) }}:</strong>
+              <p v-html="$t(`termsAndConditions.${section}.description`)" />
+            </li>
+          </ol>
+          <p>{{ $t(`termsAndConditions.have-fun`) }}</p>
+          <br />
+          <p>
+            <strong v-html="$t(`termsAndConditions.closing`)" />
+          </p>
+        </div>
+      </no-ssr>
       <div>
-        <ol>
-          <li v-for="section in sections" :key="section">
-            <strong>{{ $t(`termsAndConditions.${section}.title`) }}:</strong>
-            <p v-html="$t(`termsAndConditions.${section}.description`)" />
-          </li>
-        </ol>
-        <p>{{ $t('termsAndConditions.have-fun') }}</p>
-        <br />
-        <p>
-          <strong v-html="$t('termsAndConditions.closing')" />
-        </p>
+        <ds-modal
+          v-if="isOpen"
+          v-model="isOpen"
+          v-title="$t(`site.termsAndConditionsNewTitle`)"
+          force
+          extended
+          confirm-label="Ich habe es gelesen und verstanden"
+          cancel-label="abbrechen"
+          v-on:confirm="setNewConfirmeVersion"
+        >
+          <div>
+            <ol>
+              <li v-for="section in sections" :key="section">
+                <strong>{{ $t(`termsAndConditions.${section}.title`) }}:</strong>
+                <p v-html="$t(`termsAndConditions.${section}.description`)" />
+              </li>
+            </ol>
+            <p>{{ $t(`termsAndConditions.have-fun`) }}</p>
+            <br />
+            <p>
+              <strong v-html="$t(`termsAndConditions.closing`)" />
+            </p>
+          </div>
+        </ds-modal>
+        <div v-if="isLoggedIn">
+          <ds-button primary icon="rocket" class="display:none" @click="isOpen = true">
+            {{ $t(`site.termsAndConditionsNewConfirm`) }}
+          </ds-button>
+        </div>
       </div>
     </ds-container>
   </div>
 </template>
 
 <script>
+import { mapGetters, mapActions } from 'vuex'
+export const VERSION = '0.0.3'
+
 export default {
   layout: 'default',
   head() {
     return {
       title: this.$t('site.termsAndConditions'),
     }
   },
+  computed: {
+    ...mapGetters({
+      isLoggedIn: 'auth/isLoggedIn',
+    }),
+  },
+  mounted() {
+    if (this.isLoggedIn) {
```


Again, I would prefer to redirect the user already on the server.